Mi primera version de los estilos

EdbertAlan opened the pull request:
Aqui esta mi hoja de estilo para revision

EdbertAlan committed:
Mi primera version dew los estilos

bidkar reviewed (changes requested):
@EdbertAlan los estilos propuestos son correctos, pero puedo notar que el archivo style.css lo guardaste en la carpeta raiz del repositorio, para lo cual te solicito que lo organices en base a un proyecto web como es costumbre en nuestra empresa.

src/styles/style.css

EdbertAlan committed:
Correxiones del PR #1

Se implemento la estructura correcta del proyecto web para ubicar el
archivo de hoja de estilos en el directorio indicado.

bidkar reviewed (approved):
@EdbertAlan Correcto, el trabajo esta culminado de acuerdo a la solicitud.
